### PR TITLE
last changes almost complete flux

### PR DESCRIPTION
**Pull request by Artmo10**

### CODE REVIEW

File: input_files/464108170_985575383606857_4196013276166222587_n.jpg
Comment (diegovillarroelsandy) [inline input_files/464108170_985575383606857_4196013276166222587_n.jpg:1]: Please delete the sample files from downloaded_files folder

File: input_files/WhatsApp Video 2024-11-06 at 21.36.13.mp4
Comment (diegovillarroelsandy) [inline input_files/WhatsApp Video 2024-11-06 at 21.36.13.mp4:1]: Same here about the files

File: input_files/Form Component.jpg
Comment (diegovillarroelsandy) [inline input_files/Form Component.jpg:1]: Same here about the files inside

**Review by diegovillarroelsandy — COMMENTED** [review]
I have left some minor comments about unnecesary files in the pr

**Comment by Artmo10** [comment]
Unnecessary files were removed

**Review by diegovillarroelsandy — APPROVED** [review]

**Pull request merged**